### PR TITLE
glow: add page

### PR DESCRIPTION
**Pull request by MarkusG**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/glow.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# glow
+
+> Render Markdown in the terminal.
+> More infomration: <https://github.com/charmbracelet/glow>.
+
+- Run glow and select a file to view:
+
+`glow`
+
+- Render a Markdown file to the terminal:
```
Comment (navarroaxel) [inline pages/common/glow.md:10]: We use `markdown` in lower case:
```suggestion
- Render a markdown file to the terminal:
```

File: pages/common/glow.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`glow {{FILE}}`
```
Comment (navarroaxel) [inline pages/common/glow.md:12]:
```suggestion
`glow {{path/to/file}}`
```

File: pages/common/glow.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- View a Markdown file (paginated):
+
+`glow -p {{FILE}}`
```
Comment (navarroaxel) [inline pages/common/glow.md:16]:
```suggestion
`glow -p {{path/to/file}}`
```

File: pages/common/glow.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- Run glow and select a file to view:
+
+`glow`
+
+- Render a Markdown file to the terminal:
+
+`glow {{FILE}}`
+
+- View a Markdown file (paginated):
```
Comment (navarroaxel) [inline pages/common/glow.md:14]: Maybe this is better:
```suggestion
- Open a markdown file using a pager:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @MarkusG, welcome and thanks for your submission. I left some comments.

I like if you add examples for render from a GitHub repo and another using a URL. Could you please add these?

File: pages/common/glow.md (hunk 4 above)
Comment (MarkusG) [reply pages/common/glow.md]: What do you think of the change in 7dc2f82db1aa60e30571643c063aaece99f85846?

Comment (navarroaxel) [reply pages/common/glow.md]: Looks good.

**Comment by MarkusG** [comment]
Resolved code review and added requested examples.

File: pages/common/glow.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`glow -p {{path/to/file}}`
+
+- View a file from a URL:
+
+`glow {{https://example.com/file.md}}`
+
+- View a GitHub/GitLab README:
+
+`glow {{github.com/user/repo}}`
```
Comment (navarroaxel) [inline pages/common/glow.md:24]: Based on the `gh.md` page:
```suggestion
`glow {{github.com/owner/repository}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @MarkusG!

**Pull request merged**